### DISCUSSION TITLE
Flagged Account

### DISCUSSION THREAD

**Discussion by gddr** [community/community#27551] [Other Feature Feedback, Questions, & Ideas]
My github account has been flagged. Please let me know my violations and also please help me settle this. My apology if I did wrong action here.

**Comment by jfullstackdev** [comment]
you mean you really had a violation ? 

try Support too 

https://support.github.com/contact/account

**Comment by DSDmark** [comment]
Hi @gddr,

There are a lot of reasons to cause this issue:-

> While using GitHub, you agree that you will not under any circumstances:
- harass, abuse, threaten, or incite violence towards any individual or group, including GitHub employees, officers, and agents, or other GitHub 
- use our servers for any form of excessive automated bulk activity (for example, spamming), or relay any other form of unsolicited advertising or solicitation through our servers, such as get-rich-quick schemes;
- attempt to disrupt or tamper with GitHub's servers in ways that could harm our Website or Service, to place undue burden on GitHub's servers through automated means, or to access GitHub's Service in ways that exceed your authorization (other than those authorized by the GitHub Bug Bounty program);
- impersonate any person or entity, including any of our employees or representatives, including through false association with GitHub, or by fraudulently misrepresenting your identity or site's purpose; 
- violate the privacy of any third party, such as by posting another person's personal information without consent.

Information derived from GitHub's [Terms of Service](https://help.github.com/articles/github-terms-of-service/).

So, only one thing you can do as Mr.@jdevstatic describe contact to **github-support**
